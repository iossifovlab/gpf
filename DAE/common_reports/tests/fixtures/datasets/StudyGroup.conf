[dataset]

name = StudyGroup
id = StudyGroup

studies = Study1,Study3
description = Study Group
