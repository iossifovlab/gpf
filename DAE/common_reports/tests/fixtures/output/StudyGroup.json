{
    "families_report": {
        "families_total": 6,
        "people_counters": [
            {
                "counters": [
                    {
                        "people_male": 0,
                        "people_female": 0,
                        "people_unspecified": 0,
                        "people_total": 0,
                        "phenotype": "pheno",
                        "people_roles": [
                            "prb",
                            "sib"
                        ]
                    },
                    {
                        "people_male": 0,
                        "people_female": 2,
                        "people_unspecified": 0,
                        "people_total": 2,
                        "phenotype": "phenotype2",
                        "people_roles": [
                            "prb",
                            "sib"
                        ]
                    },
                    {
                        "people_male": 1,
                        "people_female": 5,
                        "people_unspecified": 0,
                        "people_total": 6,
                        "phenotype": "phenotype1",
                        "people_roles": [
                            "prb",
                            "sib"
                        ]
                    },
                    {
                        "people_male": 0,
                        "people_female": 0,
                        "people_unspecified": 0,
                        "people_total": 0,
                        "phenotype": "unaffected",
                        "people_roles": [
                            "prb",
                            "sib"
                        ]
                    }
                ],
                "roles": [
                    "prb",
                    "sib"
                ]
            },
            {
                "counters": [
                    {
                        "people_male": 2,
                        "people_female": 0,
                        "people_unspecified": 0,
                        "people_total": 2,
                        "phenotype": "pheno",
                        "people_roles": [
                            "mom",
                            "dad"
                        ]
                    },
                    {
                        "people_male": 1,
                        "people_female": 0,
                        "people_unspecified": 0,
                        "people_total": 1,
                        "phenotype": "phenotype2",
                        "people_roles": [
                            "mom",
                            "dad"
                        ]
                    },
                    {
                        "people_male": 0,
                        "people_female": 2,
                        "people_unspecified": 0,
                        "people_total": 2,
                        "phenotype": "phenotype1",
                        "people_roles": [
                            "mom",
                            "dad"
                        ]
                    },
                    {
                        "people_male": 3,
                        "people_female": 4,
                        "people_unspecified": 0,
                        "people_total": 7,
                        "phenotype": "unaffected",
                        "people_roles": [
                            "mom",
                            "dad"
                        ]
                    }
                ],
                "roles": [
                    "mom",
                    "dad"
                ]
            }
        ],
        "families_counters": [
            {
                "counters": [],
                "phenotype": "pheno"
            },
            {
                "counters": [],
                "phenotype": "phenotype2"
            },
            {
                "counters": [
                    {
                        "pedigree": [
                            [
                                "f1",
                                "mom1",
                                "0",
                                "0",
                                "F",
                                "#ffffff",
                                null,
                                false,
                                "",
                                ""
                            ],
                            [
                                "f1",
                                "dad1",
                                "0",
                                "0",
                                "M",
                                "#ffffff",
                                null,
                                false,
                                "",
                                ""
                            ],
                            [
                                "f1",
                                "ch1",
                                "dad1",
                                "mom1",
                                "F",
                                "#e35252",
                                null,
                                false,
                                "",
                                ""
                            ]
                        ],
                        "pedigrees_count": 2
                    },
                    {
                        "pedigree": [
                            [
                                "f6",
                                "mom6",
                                "0",
                                "0",
                                "F",
                                "#ffffff",
                                null,
                                false,
                                "",
                                ""
                            ],
                            [
                                "f6",
                                "dad6",
                                "0",
                                "0",
                                "M",
                                "#ffffff",
                                null,
                                false,
                                "",
                                ""
                            ],
                            [
                                "f6",
                                "ch6",
                                "dad6",
                                "mom6",
                                "M",
                                "#e35252",
                                null,
                                false,
                                "",
                                ""
                            ]
                        ],
                        "pedigrees_count": 1
                    }
                ],
                "phenotype": "phenotype1"
            },
            {
                "counters": [],
                "phenotype": "unaffected"
            },
            {
                "counters": [
                    {
                        "pedigree": [
                            [
                                "f4",
                                "mom4",
                                "0",
                                "0",
                                "F",
                                "#e35252",
                                null,
                                false,
                                "",
                                ""
                            ],
                            [
                                "f4",
                                "ch4",
                                "dad4",
                                "mom4",
                                "F",
                                "#e35252",
                                null,
                                false,
                                "",
                                ""
                            ],
                            [
                                "f4",
                                "ch4.1",
                                "dad4",
                                "mom4",
                                "F",
                                "#b8008a",
                                null,
                                false,
                                "",
                                ""
                            ],
                            [
                                "f4",
                                "dad4",
                                "0",
                                "0",
                                "M",
                                "#E0E0E0",
                                null,
                                true,
                                "",
                                ""
                            ]
                        ],
                        "pedigrees_count": 1
                    },
                    {
                        "pedigree": [
                            [
                                "f5",
                                "mom5",
                                "0",
                                "0",
                                "F",
                                "#e35252",
                                null,
                                false,
                                "",
                                ""
                            ],
                            [
                                "f5",
                                "dad5",
                                "0",
                                "0",
                                "M",
                                "#aaaaaa",
                                null,
                                false,
                                "",
                                ""
                            ],
                            [
                                "f5",
                                "ch5",
                                "dad5",
                                "mom5",
                                "F",
                                "#e35252",
                                null,
                                false,
                                "",
                                ""
                            ],
                            [
                                "f5",
                                "ch5.1",
                                "dad5",
                                "mom5",
                                "F",
                                "#b8008a",
                                null,
                                false,
                                "",
                                ""
                            ]
                        ],
                        "pedigrees_count": 1
                    },
                    {
                        "pedigree": [
                            [
                                "f2",
                                "mom2",
                                "0",
                                "0",
                                "F",
                                "#ffffff",
                                null,
                                false,
                                "",
                                ""
                            ],
                            [
                                "f2",
                                "dad2",
                                "0",
                                "0",
                                "M",
                                "#b8008a",
                                null,
                                false,
                                "",
                                ""
                            ],
                            [
                                "f2",
                                "ch2",
                                "dad2",
                                "mom2",
                                "F",
                                "#e35252",
                                null,
                                false,
                                "",
                                ""
                            ]
                        ],
                        "pedigrees_count": 1
                    }
                ],
                "phenotype": -1
            }
        ],
        "phenotypes": [
            "pheno",
            "phenotype2",
            "phenotype1",
            "unaffected"
        ]
    },
    "denovo_report": {
        "effect_groups": [],
        "effect_types": [
            "Frame-shift",
            "Missense"
        ],
        "phenotypes": [
            "pheno",
            "phenotype2",
            "phenotype1",
            "unaffected"
        ],
        "tables": [
            {
                "rows": [
                    {
                        "effect_type": "Frame-shift",
                        "row": [
                            {
                                "events_people_count": 0,
                                "events_people_percent": 0,
                                "events_count": 0,
                                "events_rate_per_child": 0,
                                "phenotype": "pheno",
                                "people_roles": [
                                    "prb",
                                    "sib"
                                ]
                            },
                            {
                                "events_people_count": 0,
                                "events_people_percent": 0,
                                "events_count": 0,
                                "events_rate_per_child": 0,
                                "phenotype": "phenotype2",
                                "people_roles": [
                                    "prb",
                                    "sib"
                                ]
                            },
                            {
                                "events_people_count": 1,
                                "events_people_percent": 0.16666666666666666,
                                "events_count": 1,
                                "events_rate_per_child": 0.2,
                                "phenotype": "phenotype1",
                                "people_roles": [
                                    "prb",
                                    "sib"
                                ]
                            },
                            {
                                "events_people_count": 0,
                                "events_people_percent": 0,
                                "events_count": 0,
                                "events_rate_per_child": 0,
                                "phenotype": "unaffected",
                                "people_roles": [
                                    "prb",
                                    "sib"
                                ]
                            }
                        ]
                    },
                    {
                        "effect_type": "Missense",
                        "row": [
                            {
                                "events_people_count": 0,
                                "events_people_percent": 0,
                                "events_count": 0,
                                "events_rate_per_child": 0,
                                "phenotype": "pheno",
                                "people_roles": [
                                    "prb",
                                    "sib"
                                ]
                            },
                            {
                                "events_people_count": 1,
                                "events_people_percent": 0.5,
                                "events_count": 1,
                                "events_rate_per_child": 0.2,
                                "phenotype": "phenotype2",
                                "people_roles": [
                                    "prb",
                                    "sib"
                                ]
                            },
                            {
                                "events_people_count": 4,
                                "events_people_percent": 0.6666666666666666,
                                "events_count": 4,
                                "events_rate_per_child": 0.8,
                                "phenotype": "phenotype1",
                                "people_roles": [
                                    "prb",
                                    "sib"
                                ]
                            },
                            {
                                "events_people_count": 0,
                                "events_people_percent": 0,
                                "events_count": 0,
                                "events_rate_per_child": 0,
                                "phenotype": "unaffected",
                                "people_roles": [
                                    "prb",
                                    "sib"
                                ]
                            }
                        ]
                    }
                ],
                "roles": [
                    "prb",
                    "sib"
                ]
            },
            {
                "rows": [
                    {
                        "effect_type": "Frame-shift",
                        "row": [
                            {
                                "events_people_count": 0,
                                "events_people_percent": 0,
                                "events_count": 0,
                                "events_rate_per_child": 0,
                                "phenotype": "pheno",
                                "people_roles": [
                                    "mom",
                                    "dad"
                                ]
                            },
                            {
                                "events_people_count": 0,
                                "events_people_percent": 0,
                                "events_count": 0,
                                "events_rate_per_child": 0,
                                "phenotype": "phenotype2",
                                "people_roles": [
                                    "mom",
                                    "dad"
                                ]
                            },
                            {
                                "events_people_count": 0,
                                "events_people_percent": 0,
                                "events_count": 0,
                                "events_rate_per_child": 0,
                                "phenotype": "phenotype1",
                                "people_roles": [
                                    "mom",
                                    "dad"
                                ]
                            },
                            {
                                "events_people_count": 0,
                                "events_people_percent": 0,
                                "events_count": 0,
                                "events_rate_per_child": 0,
                                "phenotype": "unaffected",
                                "people_roles": [
                                    "mom",
                                    "dad"
                                ]
                            }
                        ]
                    },
                    {
                        "effect_type": "Missense",
                        "row": [
                            {
                                "events_people_count": 0,
                                "events_people_percent": 0,
                                "events_count": 0,
                                "events_rate_per_child": 0,
                                "phenotype": "pheno",
                                "people_roles": [
                                    "mom",
                                    "dad"
                                ]
                            },
                            {
                                "events_people_count": 0,
                                "events_people_percent": 0,
                                "events_count": 0,
                                "events_rate_per_child": 0,
                                "phenotype": "phenotype2",
                                "people_roles": [
                                    "mom",
                                    "dad"
                                ]
                            },
                            {
                                "events_people_count": 0,
                                "events_people_percent": 0,
                                "events_count": 0,
                                "events_rate_per_child": 0,
                                "phenotype": "phenotype1",
                                "people_roles": [
                                    "mom",
                                    "dad"
                                ]
                            },
                            {
                                "events_people_count": 0,
                                "events_people_percent": 0,
                                "events_count": 0,
                                "events_rate_per_child": 0,
                                "phenotype": "unaffected",
                                "people_roles": [
                                    "mom",
                                    "dad"
                                ]
                            }
                        ]
                    }
                ],
                "roles": [
                    "mom",
                    "dad"
                ]
            }
        ]
    },
    "study_name": "StudyGroup",
    "phenotype": "pheno,phenotype2,phenotype1,unaffected",
    "study_type": "WE",
    "study_year": null,
    "pub_med": null,
    "families": 6,
    "number_of_probands": 6,
    "number_of_siblings": 2,
    "denovo": true,
    "transmitted": false,
    "study_description": "Study Group",
    "is_downloadable": false
}