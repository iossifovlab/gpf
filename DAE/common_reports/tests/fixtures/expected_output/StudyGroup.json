{
    "families_report": {
        "families_total": 6,
        "people_counters": [
            {
                "group_name": "Role",
                "columns": [
                    "mom",
                    "dad",
                    "prb",
                    "sib"
                ],
                "rows": [
                    "people_male",
                    "people_female",
                    "people_total"
                ],
                "counters": [
                    {
                        "people_male": 0,
                        "people_female": 6,
                        "people_total": 6,
                        "column": "mom"
                    },
                    {
                        "people_male": 6,
                        "people_female": 0,
                        "people_total": 6,
                        "column": "dad"
                    },
                    {
                        "people_male": 1,
                        "people_female": 5,
                        "people_total": 6,
                        "column": "prb"
                    },
                    {
                        "people_male": 0,
                        "people_female": 2,
                        "people_total": 2,
                        "column": "sib"
                    }
                ]
            },
            {
                "group_name": "Diagnosis",
                "columns": [
                    "pheno",
                    "phenotype1",
                    "phenotype2",
                    "unaffected"
                ],
                "rows": [
                    "people_male",
                    "people_female",
                    "people_total"
                ],
                "counters": [
                    {
                        "people_male": 2,
                        "people_female": 0,
                        "people_total": 2,
                        "column": "pheno"
                    },
                    {
                        "people_male": 1,
                        "people_female": 7,
                        "people_total": 8,
                        "column": "phenotype1"
                    },
                    {
                        "people_male": 1,
                        "people_female": 2,
                        "people_total": 3,
                        "column": "phenotype2"
                    },
                    {
                        "people_male": 3,
                        "people_female": 4,
                        "people_total": 7,
                        "column": "unaffected"
                    }
                ]
            },
            {
                "group_name": "Role and Diagnosis",
                "columns": [
                    "dad and pheno",
                    "mom and phenotype1",
                    "prb and phenotype1",
                    "sib and phenotype2",
                    "mom and unaffected",
                    "dad and unaffected",
                    "dad and phenotype2"
                ],
                "rows": [
                    "people_male",
                    "people_female",
                    "people_total"
                ],
                "counters": [
                    {
                        "people_male": 2,
                        "people_female": 0,
                        "people_total": 2,
                        "column": "dad and pheno"
                    },
                    {
                        "people_male": 0,
                        "people_female": 2,
                        "people_total": 2,
                        "column": "mom and phenotype1"
                    },
                    {
                        "people_male": 1,
                        "people_female": 5,
                        "people_total": 6,
                        "column": "prb and phenotype1"
                    },
                    {
                        "people_male": 0,
                        "people_female": 2,
                        "people_total": 2,
                        "column": "sib and phenotype2"
                    },
                    {
                        "people_male": 0,
                        "people_female": 4,
                        "people_total": 4,
                        "column": "mom and unaffected"
                    },
                    {
                        "people_male": 3,
                        "people_female": 0,
                        "people_total": 3,
                        "column": "dad and unaffected"
                    },
                    {
                        "people_male": 1,
                        "people_female": 0,
                        "people_total": 1,
                        "column": "dad and phenotype2"
                    }
                ]
            }
        ],
        "families_counters": [
            {
                "group_name": "Study phenotype",
                "phenotypes": [
                    "pheno",
                    "phenotype1",
                    "phenotype2",
                    "unaffected"
                ],
                "counters": [
                    {
                        "counters": [
                            {
                                "pedigree": [
                                    [
                                        "f1",
                                        "mom1",
                                        "0",
                                        "0",
                                        "F",
                                        "#ffffff",
                                        null,
                                        false,
                                        "",
                                        ""
                                    ],
                                    [
                                        "f1",
                                        "dad1",
                                        "0",
                                        "0",
                                        "M",
                                        "#ffffff",
                                        null,
                                        false,
                                        "",
                                        ""
                                    ],
                                    [
                                        "f1",
                                        "ch1",
                                        "dad1",
                                        "mom1",
                                        "F",
                                        "#e35252",
                                        null,
                                        false,
                                        "",
                                        ""
                                    ]
                                ],
                                "pedigrees_count": 2
                            },
                            {
                                "pedigree": [
                                    [
                                        "f6",
                                        "mom6",
                                        "0",
                                        "0",
                                        "F",
                                        "#ffffff",
                                        null,
                                        false,
                                        "",
                                        ""
                                    ],
                                    [
                                        "f6",
                                        "dad6",
                                        "0",
                                        "0",
                                        "M",
                                        "#ffffff",
                                        null,
                                        false,
                                        "",
                                        ""
                                    ],
                                    [
                                        "f6",
                                        "ch6",
                                        "dad6",
                                        "mom6",
                                        "M",
                                        "#e35252",
                                        null,
                                        false,
                                        "",
                                        ""
                                    ]
                                ],
                                "pedigrees_count": 1
                            }
                        ]
                    },
                    {
                        "counters": [
                            {
                                "pedigree": [
                                    [
                                        "f2",
                                        "mom2",
                                        "0",
                                        "0",
                                        "F",
                                        "#ffffff",
                                        null,
                                        false,
                                        "",
                                        ""
                                    ],
                                    [
                                        "f2",
                                        "dad2",
                                        "0",
                                        "0",
                                        "M",
                                        "#b8008a",
                                        null,
                                        false,
                                        "",
                                        ""
                                    ],
                                    [
                                        "f2",
                                        "ch2",
                                        "dad2",
                                        "mom2",
                                        "F",
                                        "#e35252",
                                        null,
                                        false,
                                        "",
                                        ""
                                    ]
                                ],
                                "pedigrees_count": 1
                            }
                        ]
                    },
                    {
                        "counters": [
                            {
                                "pedigree": [
                                    [
                                        "f4",
                                        "mom4",
                                        "0",
                                        "0",
                                        "F",
                                        "#e35252",
                                        null,
                                        false,
                                        "",
                                        ""
                                    ],
                                    [
                                        "f4",
                                        "ch4",
                                        "dad4",
                                        "mom4",
                                        "F",
                                        "#e35252",
                                        null,
                                        false,
                                        "",
                                        ""
                                    ],
                                    [
                                        "f4",
                                        "ch4.1",
                                        "dad4",
                                        "mom4",
                                        "F",
                                        "#b8008a",
                                        null,
                                        false,
                                        "",
                                        ""
                                    ],
                                    [
                                        "f4",
                                        "dad4",
                                        "0",
                                        "0",
                                        "M",
                                        "#E0E0E0",
                                        null,
                                        true,
                                        "",
                                        ""
                                    ]
                                ],
                                "pedigrees_count": 1
                            },
                            {
                                "pedigree": [
                                    [
                                        "f5",
                                        "mom5",
                                        "0",
                                        "0",
                                        "F",
                                        "#e35252",
                                        null,
                                        false,
                                        "",
                                        ""
                                    ],
                                    [
                                        "f5",
                                        "dad5",
                                        "0",
                                        "0",
                                        "M",
                                        "#aaaaaa",
                                        null,
                                        false,
                                        "",
                                        ""
                                    ],
                                    [
                                        "f5",
                                        "ch5",
                                        "dad5",
                                        "mom5",
                                        "F",
                                        "#e35252",
                                        null,
                                        false,
                                        "",
                                        ""
                                    ],
                                    [
                                        "f5",
                                        "ch5.1",
                                        "dad5",
                                        "mom5",
                                        "F",
                                        "#b8008a",
                                        null,
                                        false,
                                        "",
                                        ""
                                    ]
                                ],
                                "pedigrees_count": 1
                            }
                        ]
                    }
                ],
                "legend": [
                    {
                        "id": "phenotype1",
                        "name": "phenotype 1",
                        "color": "#e35252"
                    },
                    {
                        "id": "phenotype2",
                        "name": "phenotype 2",
                        "color": "#b8008a"
                    },
                    {
                        "id": "phenotype3",
                        "name": "phenotype 3",
                        "color": "#e3d252"
                    },
                    {
                        "id": "unaffected",
                        "name": "unaffected",
                        "color": "#ffffff"
                    },
                    {
                        "id": "unknown",
                        "name": "unknown",
                        "color": "#aaaaaa"
                    },
                    {
                        "id": "missing-person",
                        "name": "missing-person",
                        "color": "#E0E0E0"
                    }
                ]
            }
        ]
    },
    "denovo_report": {
        "tables": [
            {
                "rows": [
                    {
                        "effect_type": "Frame-shift",
                        "row": [
                            {
                                "number_of_observed_events": 1,
                                "number_of_children_with_event": 1,
                                "observed_rate_per_child": 0.125,
                                "percent_of_children_with_events": 0.125,
                                "column": "prb"
                            },
                            {
                                "number_of_observed_events": 0,
                                "number_of_children_with_event": 0,
                                "observed_rate_per_child": 0.0,
                                "percent_of_children_with_events": 0.0,
                                "column": "sib"
                            }
                        ]
                    },
                    {
                        "effect_type": "Missense",
                        "row": [
                            {
                                "number_of_observed_events": 4,
                                "number_of_children_with_event": 4,
                                "observed_rate_per_child": 0.5,
                                "percent_of_children_with_events": 0.5,
                                "column": "prb"
                            },
                            {
                                "number_of_observed_events": 1,
                                "number_of_children_with_event": 1,
                                "observed_rate_per_child": 0.125,
                                "percent_of_children_with_events": 0.125,
                                "column": "sib"
                            }
                        ]
                    }
                ],
                "group_name": "Role",
                "columns": [
                    "prb",
                    "sib"
                ],
                "effect_groups": [],
                "effect_types": [
                    "Frame-shift",
                    "Missense"
                ]
            },
            {
                "rows": [
                    {
                        "effect_type": "Frame-shift",
                        "row": [
                            {
                                "number_of_observed_events": 1,
                                "number_of_children_with_event": 1,
                                "observed_rate_per_child": 0.125,
                                "percent_of_children_with_events": 0.125,
                                "column": "phenotype1"
                            },
                            {
                                "number_of_observed_events": 0,
                                "number_of_children_with_event": 0,
                                "observed_rate_per_child": 0.0,
                                "percent_of_children_with_events": 0.0,
                                "column": "phenotype2"
                            }
                        ]
                    },
                    {
                        "effect_type": "Missense",
                        "row": [
                            {
                                "number_of_observed_events": 4,
                                "number_of_children_with_event": 4,
                                "observed_rate_per_child": 0.5,
                                "percent_of_children_with_events": 0.5,
                                "column": "phenotype1"
                            },
                            {
                                "number_of_observed_events": 1,
                                "number_of_children_with_event": 1,
                                "observed_rate_per_child": 0.125,
                                "percent_of_children_with_events": 0.125,
                                "column": "phenotype2"
                            }
                        ]
                    }
                ],
                "group_name": "Diagnosis",
                "columns": [
                    "phenotype1",
                    "phenotype2"
                ],
                "effect_groups": [],
                "effect_types": [
                    "Frame-shift",
                    "Missense"
                ]
            },
            {
                "rows": [
                    {
                        "effect_type": "Frame-shift",
                        "row": [
                            {
                                "number_of_observed_events": 1,
                                "number_of_children_with_event": 1,
                                "observed_rate_per_child": 0.125,
                                "percent_of_children_with_events": 0.125,
                                "column": "prb and phenotype1"
                            },
                            {
                                "number_of_observed_events": 0,
                                "number_of_children_with_event": 0,
                                "observed_rate_per_child": 0.0,
                                "percent_of_children_with_events": 0.0,
                                "column": "sib and phenotype2"
                            }
                        ]
                    },
                    {
                        "effect_type": "Missense",
                        "row": [
                            {
                                "number_of_observed_events": 4,
                                "number_of_children_with_event": 4,
                                "observed_rate_per_child": 0.5,
                                "percent_of_children_with_events": 0.5,
                                "column": "prb and phenotype1"
                            },
                            {
                                "number_of_observed_events": 1,
                                "number_of_children_with_event": 1,
                                "observed_rate_per_child": 0.125,
                                "percent_of_children_with_events": 0.125,
                                "column": "sib and phenotype2"
                            }
                        ]
                    }
                ],
                "group_name": "Role and Diagnosis",
                "columns": [
                    "prb and phenotype1",
                    "sib and phenotype2"
                ],
                "effect_groups": [],
                "effect_types": [
                    "Frame-shift",
                    "Missense"
                ]
            }
        ]
    },
    "study_name": "StudyGroup",
    "phenotype": ["pheno", "phenotype2", "phenotype1", "unaffected"],
    "study_type": "WE",
    "study_year": null,
    "pub_med": null,
    "families": 6,
    "number_of_probands": 6,
    "number_of_siblings": 2,
    "denovo": true,
    "transmitted": false,
    "study_description": "Study Group",
    "is_downloadable": false
}